wiki/configuration.md

```diff
@@ -296,3 +303,13 @@ Eiffel-Intelligence will decrypt password properties with help of the provided s
 Execute Eiffel-Intelligence with "jasypt.encryptor.password=mySecretEncryptorPassword" flag, example:
 
         java -jar eiffel-intelligence-<version>.war --jasypt.encryptor.password=mySecretEncryptorPassword --spring.config.location=/path/to/application.properties
+
+## Removal of unused bindings from RabbitMQ using mongoDB
```

Comment:
Is git messing with me or is this text duplicated? Remove the paragraph at the bottom and keep the first one.